package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "flycomponents",
-  "version": "3.2.5",
+  "version": "3.2.7",
```

Comment:
Not a show stopper but we are missing 3.2.6. 

What I use to do when I need to change the code after generating the version is:
1. Delete latest two commits (tag and documentation): `git reset --hard HEAD~2`
2. Delete version remotely: `git push --delete origin <tag>`
3. Delete version remotely: `git tag -d <tag>
4. Generate version again: `npm version patch`

As I mentioned is not a show stopper because it is not the first time it happens but it would be nice to have all versions.